packages/neuron-wallet/src/controllers/api.ts

```diff
@@ -58,6 +59,11 @@ export default class ApiController {
   private registerHandlers() {
     const handle = this.handleChannel
 
+    // sync messages
+    ipcMain.on('get-locale', e => {
```

Comment:
Can we use `handle` here instead, so that it is consistent with the rest of the invoking APIs?

Reply:
> Can we use `handle` here instead, so that it is consistent with the rest of the invoking APIs?

We could refactor it later, I used synchronous message here to minimize the update on messages cuz `app.getLocale` is synchronous.